packages/dev/core/src/assetContainer.ts

```diff
@@ -310,9 +313,9 @@ export class AssetContainer extends AbstractScene {
                 }
             }
 
-            clone.position = source.position.clone();
-            clone.rotation = source.rotation.clone();
-            clone.scaling = source.scaling.clone();
+            clone.position = source.position;
+            clone.rotation = source.rotation;
+            clone.scaling = source.scaling;

```

Comment:
You should not even need to do this, the position/rotation/scaling properties of the clone should already have the right values. On the contrary, setting the same position/rotation/scaling objects as the source will link the clone and the source: when you update the values of one, the other will be updated too as there are the same objects.

Reply:
If I don't set it through, the positions aren't updated correctly. Maybe it's because the asset container is cloning to a different scene?

Reply:
That's because instantiateModels uses `instantiateHierarchy` currently and that does copy the positions and scaling: https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Meshes/mesh.ts#L743

Reply:
I have just rechecked and it does work without the 3 lines of code, in both the clone and instance mode. Can you recheck on your end?

For me, the code in `instantiateHierarchy` is not necessary either.

Reply:
My bad, we need to copy + clone when the source mesh is in fact an instance.

Reply:
Is this resolved?